src/features/session_trace/aggregate/index.js

```diff
@@ -189,16 +193,16 @@ export class Aggregate extends AggregateBase {
   }
 
   #prepareHarvest (options) {
-    /* Standalone refers to the legacy version of ST before the idea of 'session' or the Replay feature existed.
-      It has a different behavior on returning a payload for harvest than when used in tandem with either of those concepts. */
     if (this.isStandalone) {
-      if (now() > MAX_TRACE_DURATION) { // been collecting for over the longest duration we should run for, empty trace object so ST has nothing to send
-        this.#scheduler.stopTimer()
-        this.trace = {}
+      if (this.ptid && now() >= MAX_TRACE_DURATION) {
```

Comment:
(In practice this isn't an actual issue, but if the purpose is to last send buffered data, `ptid` isn't really required. Though edge & theoretical case, `#prepareHarvest` does apply to the first `runHarvest` call too, wherein `ptid` is undefined and subsequently fetched.)